src/test/java/EulerTwoTest.java

```diff
@@ -0,0 +1,72 @@
+import euler2.EulerTwo;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EulerTwoTest {
+    private EulerTwo eulerTwo;
+
+    @BeforeEach
+    void setUp() {
+        eulerTwo = new EulerTwo();
+    }
+
+    @Test
+    void shouldReturn10ForLimit9(){
+        //given
+        int limit = 9;
+
+        //when
+        int actualSum = eulerTwo.sumFibonacciEvenNumbers(limit);
+
+        //then
+        int expectedSum = 10;
+        assertEquals(expectedSum, actualSum);
+    }
+
+    @Test
+    void shouldNotReturn10ForLimit8(){
```

Comment:
Problem z tym testem jest taki że kod zwraca Ci 10 mimo, że nie powinien. Nie chcę Ci od razu podsuwać rozwiązania - postaraj się znaleźć sama gdzie jest błąd. Swoją drogą - chyba widzisz, dlaczego testy są przydatne :)

Reply:
już wczoraj się domyślałam co jest źle ale nie wiedziałam jak poprawić, teraz się przedebugowałam i już wiem, kolejność w pętli jest zła, na razie dodałam takiego brzydkiego potworka na którego wpadłam tak na szybko, może później wymyślę jak zrobić to lepiej, póki co działa :D

Reply:
podpowiedź jesr również mile widziana ;)

Reply:
poprawione!